docs/docs/smart-contracts/access-control/Extensions/enumerable.md

```diff
@@ -0,0 +1,84 @@
+---
+sidebar_position: 1
```

Comment:
maybe it will be better if you set position with __category_.json_ file, as well as we do it with other extensions)